src/jit/emitxarch.cpp

```diff
@@ -4148,6 +4148,29 @@ void emitter::emitIns_AR(instruction ins, emitAttr attr, regNumber base, int off
     emitCurIGsize += sz;
 }
 
+void emitter::emitIns_AR_R_R(
+    instruction ins, emitAttr attr, regNumber op2Reg, regNumber op3Reg, regNumber base, int offs)
+{
```

Comment:
We need method header - doc for new method - commenting again since github has hidden my previous comment

Reply:
Fixed.